e2e-tests/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
```

Comment:
reminder - let's make this a node script so we stay windows friendly before we dive deeper into bash land